test/face/scan/test_scan.py

```diff
@@ -0,0 +1,476 @@
+    """
+    qtbot.mouseClick(scantest.arti_back_wipe, Qt.LeftButton)
+
+    """
+    Confirm if the user interface elements change accordingly
+    """
+    assert scantest.arti_type.currentText() == "None"
+    assert scantest.arti_levl.currentText() == "None"
+    assert scantest.arti_rare.currentText() == "Star 0"
+    assert scantest.arti_type_name.text() == "None"
+    for item in ["main", "a", "b", "c", "d"]:
+        assert getattr(scantest, f"arti_name_{item}").currentText() == "None"
+        assert getattr(scantest, f"arti_data_{item}").text() == ""
+
+
+@pytest.mark.parametrize(
+    "_",
+    [
+        pytest.param(None, id="face.scan.rule: Test OCR functionality")
+    ]
+)
+def test_scan_arti_load(scantest, qtbot, mocker, _) -> None:
+    """
+    Test OCR functionality
+
+    :return:
+    """
+
+    """
+    Create the tesseract training data
+    """
+    mocker.patch("gi_loadouts.conf.data_prefix", f"{uuid4().hex.upper()[0:8]}-")
+    make_temp_file()
+
+    """
+    Perform the action of loading the artifact information
+    """
+    temp_prm = ""
+    tempfile = "test/static/img/gi-loadouts-ocr-test.webp"
+    savefile = str(Path(gettempdir()) / "gi-loadouts-ocr-test.webp")
+    with open(tempfile, "rb") as src_file:
+        with open(savefile, "wb") as dest_file:
+            dest_file.write(src_file.read())
+    mocker.patch.object(QFileDialog, "getOpenFileName", return_value=(savefile, temp_prm))
+    qtbot.mouseClick(scantest.arti_cnvs_load, Qt.LeftButton)
+
+    """
+    Confirm if the user interface elements change accordingly
+    """
+    def check_label():
+        """
+        Checking the interface elements change asynchronously
+        """
+        assert scantest.arti_text.text() == "Browse your local storage to load a high quality screenshot of your artifact and the statistics will automatically be computed from there."
+        assert scantest.arti_type.currentText() == "Shimenawa's Reminiscence"
+        assert scantest.arti_levl.currentText() == "Level 20"
+        assert scantest.arti_rare.currentText() == "Star 5"
+        assert scantest.arti_type_name.text() == "Morning Dew's Moment"
+        assert scantest.arti_name_main.currentText() == "Energy Recharge"
+        assert scantest.arti_data_main.text() == "51.8"
+        assert scantest.arti_name_a.currentText() == "Elemental Mastery"
+        assert scantest.arti_data_a.text() == "23.0"
+        assert scantest.arti_name_b.currentText() == "Crit Rate"
+        assert scantest.arti_data_b.text() == "6.6"
+        assert scantest.arti_name_c.currentText() == "Crit DMG"
+        assert scantest.arti_data_c.text() == "21.0"
+        assert scantest.arti_name_d.currentText() == "HP %"
+        assert scantest.arti_data_d.text() == "13.4"
+    qtbot.waitUntil(check_label)
+
+    """
+    Clear the copied file from temp directory
+    """
+    kill_temp_file()
+    if path.exists(savefile):
+        remove(savefile)
```

Comment:
As the `NamedTemporaryFile`'s path would be reasonably randomized, we need not look for whether the path exists.

```suggestion
    remove(savefile)
```